src/Raven.Server/Documents/QueueSink/KafkaQueueSink.cs

```diff
@@ -31,8 +31,8 @@ protected override IQueueSinkConsumer CreateConsumer()
 
         if (settings.UseRavenCertificate && certificateHolder?.ClientCertificate != null)
         {
-            consumerConfig.SslCertificatePem = ExportAsPem(new PemObject("CERTIFICATE", certificateHolder.ClientCertificate.RawData));
-            consumerConfig.SslKeyPem = ExportAsPem(certificateHolder.PrivateKey.Key);
+            consumerConfig.SslCertificatePem = certificateHolder.ClientCertificate.ExportCertificatePem();
+            consumerConfig.SslKeyPem = (certificateHolder.PrivateKey as RSA).GetExportableRsaPrivateKey().ExportRSAPrivateKeyPem();
```

Comment:
`ECDsa`

Reply:
It is a separate issue, this one is a 1:1 transition